pkg/service/job_service.go

```diff
@@ -0,0 +1,22 @@
+package service
+
+import (
+	"log"
+
+	"github.com/b-open/jobbuzz/internal/config"
+	"github.com/b-open/jobbuzz/pkg/model"
+)
+
+func GetJobs() []model.Job {
+	db := config.GetDb()
+
+	var jobs []model.Job
+
+	results := db.Find(&jobs)
+
+	if err := results.Error; err != nil {
+		log.Fatal(err)
```

Comment:
Return err

Also check whether empty result counts as an error, if it does we will need to consider how to handle it. E.g. return empty slice